mapviz_plugins/src/textured_marker_plugin.cpp

```diff
@@ -197,7 +200,7 @@ void TexturedMarkerPlugin::ProcessMarker(const marti_visualization_msgs::msg::Te
       markerData.expire_time = rclcpp::Time::max();
     } else {
       // Temporarily add 5 seconds to fix some existing markers.
-      markerData.expire_time = rclcpp::Time() + lifetime + rclcpp::Duration(5);
+      markerData.expire_time = rclcpp::Time() + lifetime + rclcpp::Duration(5ns);
```

Comment:
The original functionality was 5 seconds, not 5 nanoseconds.

Reply:
And it was working as intended? Interesting, since that constructor takes a number of nanoseconds. Either way, I can fix that.